SignalServiceKit/src/Messages/OWSOutgoingReceiptManager.m

```diff
@@ -202,6 +203,11 @@ - (void)process {
                 failure:^(NSError *error) {
                     OWSLogError(@"Failed to send %@ receipts to sender with error: %@", receiptName, error);
 
+                    if (error.domain == OWSSignalServiceKitErrorDomain
+                        && error.code == OWSErrorCodeNoSuchSignalRecipient) {
+                        [self dequeueReceiptsWithRecipientId:recipientId timestamps:timestamps receiptType:receiptType];
```

Comment:
Is it reasonable that we're still rejecting the promise in this case? It seems like it doesn't really make a difference based on how it's currently being used, so TIOLI.

Reply:
Right.